kind/actions/cluster_actions.go

```diff
@@ -179,7 +179,7 @@ func KubeadmInit(clusterName, version string) error {
 	}
 
 	// save the kubeconfig on the host with the loadbalancer endpoint
-	hostPort, err := getLoadBalancerPort(allNodes)
+	_, hostPort, err := GetLoadBalancerHostAndPort(allNodes)
```

Comment:
hostPort -> port?

Reply:
maybe? I don't really see a huge benefit in this rename. I recognize host is a bit inaccurate technically but it may help to think about each container as a host.

Reply:
ah i refer to host as docker host in other places, i will clarify, good point